django/http/request.py

```diff
@@ -402,6 +402,19 @@ def __init__(self, query_string=None, mutable=False, encoding=None):
                                 value)
         self._mutable = mutable
 
+    @classmethod
+    def fromkeys(cls, iterable, value='', mutable=False, encoding=None):
+        """
+        Returns a new QueryDict with keys from iterable and values equal to value.
+        Note: repeated keys are allowed.
+        """
+        q = cls('', mutable=True, encoding=encoding)
+        for key in iterable:
```

Comment:
Are you suggesting that it should be checked for explicitly?  The behaviour is currently the same for `dict.fromkeys(0)` which would raise `TypeError: 'int' object is not iterable`, a perfectly acceptable result I think.  Happy to add the test if you want, but didn't see that as necessary.